### PR TITLE
chore: publish v1.0.0

### PR DESCRIPTION
**Pull request by dshukertjr**

### CODE REVIEW

**Comment by Vinzent03** [comment]
Despite the failing test: LGTM

**Comment by dshukertjr** [comment]
Wow, what happened! I will look into it.

**Comment by dshukertjr** [comment]
Ah, it seems like a race condition within the test cases. [This](https://github.com/supabase-community/supabase-flutter/issues/236) hopefully can fix it.

File: test/client_test.dart [hunk 1]
```diff
@@ -196,12 +196,12 @@ void main() {
 
     test('Update user', () async {
       final response = await client.updateUser(
-        UserAttributes(data: {
-          'hello': 'world',
```
Comment (Vinzent03) [inline test/client_test.dart:200]: This line is deleted, but required for the test.

**Review by Vinzent03 — CHANGES_REQUESTED** [review]

**Comment by dshukertjr** [comment]
Ah, never mind. I pushed code that I didn't mean to push. Will fix it right now.

File: test/client_test.dart (hunk 1 above)
Comment (dshukertjr) [reply test/client_test.dart:200]: Yup, thanks for spotting it!

**Review by Vinzent03 — APPROVED** [review]

**Pull request merged**